.gitignore

```
@@ -6,3 +6,4 @@ _version.py
 .idea/
 venv/
 tmp/
+.vscode/settings.json
```

note that this can also be set in your global .git configuration


with this

```
[core]
	excludesfile = ~/.gitignore_global
```
and 

```
➜  ~ cat .gitignore_global 
venv/
.DS_Store
.idea/
.vscode/
```
```suggestion
.vscode/
```

however, I will allow it since `.idea/` is somehow already in here.